libdevcore/FileSignal.h

```diff
@@ -41,10 +41,21 @@ class FileSignal
             {
                 boost::filesystem::remove(_file);  // Just call once, even _f() exception happens
                 _f();
+
+                if (boost::filesystem::exists(_file))
+                {
+                    boost::filesystem::remove(_file);  // Delete file signal generated during f() is
+                                                       // executing
+                }
             }
         }
         catch (std::exception _e)
         {
+            if (boost::filesystem::exists(_file))
+            {
+                boost::filesystem::remove(_file);  // Delete file signal generated during f() is
```

Comment:
same as before, I advise to not put a comment after code

Reply:
done